public/docs/ts/latest/guide/setup.jade

```diff
@@ -0,0 +1,142 @@
+block includes
+  include ../_util-fns
+
+:marked
+  # Trying Angular
+
+  Live coding [Quickstart](../quickstart.html) in the browser is a great way to explore Angular.
+  Links on almost every documentation page will open completed samples in the browser where you can
+  play with the code, share your changes with friends, and download and run the code on your own machine.
+
+  As much fun as this is
+  * you can't ship your app in plunker
+  * you aren't always online when writing code
+  * transpiling TypeScript in the browser is slow
+  * the type support, refactoring, and code completion only work in your local IDE

```

Comment:
I've been following the writing guidelines (will share link in slack, but I think you have it). They say to capitalize. There does seem to be some leeway, but I haven't run into a situation like this where each point completes a sentence. Here's what I've been referencing: 

> 

> Punctuation and capitalization

- For numbered, lettered, and bulleted lists, in most contexts, start each list item with a capital letter and end each list item with a period.
- If a list item consists entirely of a literal string (such as a keyword), don't capitalize it or put a period after it.
- In description lists, in most contexts, start each term with a capital letter, but don't end the term with a period. In other words, for the "dt"/"term" elements of a description list, leave off the period at the end of each item. Do generally put periods at the ends of the "dd"/"description" elements.
- But there's substantial variation in appropriate capitalization and punctuation, depending on content. Don't worry too much about adhering to this guideline.